package.json

```diff
@@ -1,11 +1,9 @@
 {
-  "name": "@first-lego-league/scoring",
+  "name": "@first-lego-league/scoresheet-builder",
   "version": "0.7.7",
```

Comment:
Bump version

Reply:
is it matter? we don't publish this package.
I change the name to prevent duplication with the scoring module